app/containers/Network/containers/BulkEditAccessPoints/index.js

```diff
@@ -0,0 +1,330 @@
+    const locationId = parseInt(id, 10);
+    getChildNodes([locationId]);
+  }, [data, id]);
+
+  useEffect(() => {
+    fetchFilterEquipmentBulkEditAps(selectedLocationIds);
+  }, [selectedLocationIds]);
+
+  const getRadioDetailsByType = (radioDetails, type, radioType) => {
+    if (type === 'cellSize') {
+      const cellSize = radioDetails.radioMap[radioType].rxCellSizeDb.value;
+      return cellSize;
+    }
+    if (type === 'probeResponseThreshold') {
+      const probeResponseThreshold =
+        radioDetails.radioMap[radioType].probeResponseThresholdDb.value;
+      return probeResponseThreshold;
+    }
+    if (type === 'clientDisconnectThreshold') {
+      const clientDisconnectThreshold =
+        radioDetails.radioMap[radioType].clientDisconnectThresholdDb.value;
+      return clientDisconnectThreshold;
+    }
+    if (type === 'snrDrop') {
+      const snrDrop = radioDetails.advancedRadioMap[radioType].bestApSettings.dropInSnrPercentage;
+      return snrDrop;
+    }
+    // if (type === 'minLoad') {
+    const minLoad = radioDetails.advancedRadioMap[radioType].bestApSettings.minLoadFactor;
+    return minLoad;
+    // }
+  };
+
+  const getRadioDetails = (radioDetails, type) => {
+    const is5GHzU = getRadioDetailsByType(radioDetails, type, IS_5GHZU);
+    const is5GHzL = getRadioDetailsByType(radioDetails, type, IS_5GHZL);
+    const is2dot4GHz = getRadioDetailsByType(radioDetails, type, IS_2DOT4GHZ);
+
+    return {
+      is5GHzU,
+      is5GHzL,
+      is2dot4GHz,
+    };
+  };
+
+  const setAccessPointsBulkEditTableData = (dataSource = []) => {
+    const tableData = dataSource.items.map(({ id: key, name, channel, details }) => {
+      const cellSizeDetails = Object.values(getRadioDetails(details, 'cellSize'));
+      const probeResponseThresholdDetails = Object.values(
+        getRadioDetails(details, 'probeResponseThreshold')
+      );
+      const clientDisconnectThresholdDetails = Object.values(
+        getRadioDetails(details, 'clientDisconnectThreshold')
+      );
+      const snrDropDetails = Object.values(getRadioDetails(details, 'snrDrop'));
+      const minLoadDetails = Object.values(getRadioDetails(details, 'minLoad'));
+      return {
+        key,
+        id: key,
+        name,
+        channel,
+        cellSize: cellSizeDetails,
+        probeResponseThreshold: probeResponseThresholdDetails,
+        clientDisconnectThreshold: clientDisconnectThresholdDetails,
+        snrDrop: snrDropDetails,
+        minLoad: minLoadDetails,
+      };
+    });
+    return tableData;
+  };
+
+  const editedRowsArr = [];
+  const handleSaveChanges = updatedRows => {
+    if (updatedRows.length > 0) {
+      updatedRows.map(
+        ({
+          id: equipmentId,
+          channel,
+          cellSize,
+          probeResponseThreshold,
+          clientDisconnectThreshold,
+          snrDrop,
+          minLoad,
+        }) => {
+          const tempObj = {
+            equipmentId,
+            perRadioDetails: {
```

Comment:
As I said above radios can be any combination per AP, this and the inputs must be dynamic per AP row